hooks/declarative-subsequent-scans/templates/NOTES.txt

```diff
@@ -10,4 +10,4 @@ $ kubectl get cascadingrules
 You need to explicitly turn on scan cascading for every scan you use.
 You can do that by setting a label selector which matches all rules you want to use.
 
-Find out more, on the docs: TODO(https://github.com/secureCodeBox/secureCodeBox-v2/issues/46)
```

Comment:
We should not update this link because this would break the link (rerouting it to https://github.com/secureCodeBox/secureCodeBox/pull/46)

Reply:
Oh good catch 👍
Will fix post merge